internal/services/compute/linux_virtual_machine_scale_set_other_resource_test.go

```diff
@@ -2465,3 +2517,97 @@ resource "azurerm_linux_virtual_machine_scale_set" "test" {
 }
 `, r.template(data), data.RandomInteger)
 }
+
+func (r LinuxVirtualMachineScaleSetResource) otherSecureBootEnabled(data acceptance.TestData, secureBootEnabled bool) string {
+	return fmt.Sprintf(`
+%s
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_linux_virtual_machine_scale_set" "test" {
+  name                = "acctestvmss-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  sku                 = "Standard_DS3_V2"
```

Comment:
this is a machine with 4 cores and 14GB of ram.. an F1 will be more than sufficient here